tests/test_setup.py

```diff
@@ -175,6 +183,99 @@ def test_validate_platform_config(self):
             assert 'platform_conf' in self.hass.config.components
             assert not config['platform_conf']  # empty
 
+    def test_validate_platform_config_2(self):
+        """Test component PLATFORM_SCHEMA_BASE prio over PLATFORM_SCHEMA."""
+        platform_schema = PLATFORM_SCHEMA.extend({
+            'cheers': str,
+            'hello': str,
+        })
+        platform_schema_base = PLATFORM_SCHEMA.extend({
+            'hello': str,
```

Comment:
Maybe it would make more sense to validate the value specifically here, so that we can test that a single key fails validation later? The real platform base schema will allow extra in the source, so this test isn't intuitive for me.

Reply:
I agree the test is not intuitive, but I'm not sure I understand how you propose to change it.
Do you mean something like this:
```python
platform_schema = PLATFORM_SCHEMA.extend({
    'cheers': str,
    'hello': str,
})
platform_schema_base = PLATFORM_SCHEMA_BASE.extend({
    'hello': vol.All(str, vol.In(['world'])
})
```